Funcky/Extensions/ObsoleteDictionaryExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using System;
```

Comment:
We could also create a folder for all obsolete APIs and replicate the directory structure there, what do you think about that?

Reply:
Adressed in 6649378beb475e6efdd099a6d430cc3bc57c7d36
Filename and classnames are congruent again.